tests/rptest/tests/read_replica_e2e_test.py

```diff
@@ -143,4 +180,34 @@ def test_simple_end_to_end(self, partition_count: int,
 
         # Consume from read replica topic and validate
         self.start_consumer()
-        self.run_validation()
+        self.run_validation()  # calls self.consumer.stop()
+        assert self.redpanda
+        self.redpanda.stop()
+
+        # Run consumer again, this time with source cluster stopped.
+        # Now we can test that replicas do not write to s3.
+        self.start_consumer()
+
+        # Assert zero bytes written for at least 20 seconds.
+        total_bytes = lambda: self._bucket_usage().total_bytes
+        assert self.redpanda and self.redpanda.logger
+        er = ExpectRate(total_bytes, self.redpanda.logger)
+        zero_growth = RateTarget(max_total_sec=60,
+                                 target_sec=20,
+                                 target_min_rate=0,
+                                 target_max_rate=0)
+        er.expect_rate(zero_growth)
+
+        # Get current bucket usage
+        pre_usage = self._bucket_usage()
+        self.logger.info(f"pre_usage {pre_usage}")
+
+        # Let replica consumer run to completion, assert no s3 writes
+        self.run_consumer_validation()
```

Comment:
I'm curious, do read replicas work at all with consumer groups? I imagine not, especially if the main redpanda cluster is offline.